app/views/core/SubscribeModal.coffee

```diff
@@ -92,6 +93,14 @@ module.exports = class SubscribeModal extends ModalView
     super()
     # TODO: does this work?
     @playSound 'game-menu-open'
+    if @basicProduct and @subModalContinue
+      if @subModalContinue is 'monthly'
+        @subModalContinue = null
+        @onClickPurchaseButton()
+      else if @subModalContinue is 'lifetime'
+        @subModalContinue = null
+        unless @basicProduct.isRegionalSubscription()
+          @onClickStripeLifetimeButton()
```

Comment:
Doesn't need to block, but keep in mind that calling click handlers in code introduces noise into analytics events that track when people actually click that button. Ideally we'd differentiate between the two, or just not log an event for fake clicks.

Reply:
Leaving as is for now to capture user experience rather than explicit clicks.  We have a unique start event, in case we want to exclude these from real clicks.